addons/hr_holidays/models/hr_employee.py

```diff
@@ -194,6 +194,18 @@ def write(self, values):
 class HrEmployeePrivate(models.Model):
     _inherit = 'hr.employee'
 
+    def write(self, vals):
+        res = super().write(vals)
+        if 'resource_calendar_id' in vals:
```

Comment:
We could also check if the resource calendar is not `False` in the `vals` before doing that. 🙂

Reply:
Changes Done